maas/client/viscera/tests/test_machines.py

```diff
@@ -322,6 +323,16 @@ def test__release_with_wait_failed_disk_erasing(self):
             wait_interval=0.1
         )
 
+    def test__get_details(self):
+        machine = make_origin().Machine({
+            "system_id": make_name_without_spaces("system-id"),
+            "hostname": make_name_without_spaces("hostname"),
+        })
+        data = machine.get_details()
+        assert all('lldp', 'lshw') in data.keys()
```

Comment:
mmmmm... not sure it does the same, I just want to check lldp and lshw are keys there.